plugins/modules/device_configs_backup_workflow_manager.py

```diff
@@ -300,15 +300,42 @@ def get_device_ids_list(self):
         )
         self.log("Response collected from the API 'get_device_list' is {0}".format(str(response)), "DEBUG")
         device_list = response.get("response")
-
         self.log("Length of the device list fetched from the API 'get_device_list' is {0}".format(str(device_list)), "INFO")
-        if len(device_list) == 0:
+        original_dev_len = len(device_list)
+        if original_dev_len == 0:
             msg = "Couldn't find any devices in the inventory that match the given parameters."
             self.log(msg, "CRITICAL")
             self.module.fail_json(msg=msg)
 
-        dev_id_list = [id.get("id") for id in device_list]
+        changed_dev_list = []
+        for dev_info in device_list:
+            if dev_info.get("collectionStatus") != "Managed":
```

Comment:
Can we take an ip_address first before if check and use it in log message?
`            ip_address = device.get("managementIpAddress")
`